tests/win_test.go

```diff
@@ -0,0 +1,370 @@
+	winrmCli            = "winrmcli"
+	winrmCliCmd         = "winrm-cli"
+	namespace           = "NamespaceTestDefault"
+)
+
+func getArgument(vm_name string, add_pvc bool) []string {
+	argument := []string{"NAME=" + vm_name}
+	if add_pvc {
+		argument = append(argument, "PVCNAME="+vm_name+"-pvc")
+	}
+	return argument
+}
+
+func isVmiStarted(vmiName string) bool {
+	//ToDo rewrite using API
+	const maxWaitIterations = 60 // two minutes
+	result := false
+	for i := 0; i < maxWaitIterations; i++ {
+		phaseStatus, _, err := ktests.RunCommandWithNS(tframework.NamespaceTestDefault, "oc", "get", "vmi", "-o=go-template='{{ .status.phase }}'", vmiName)
+		Expect(err).ToNot(HaveOccurred())
+		if phaseStatus == "'Running'" {
+			result = true
+			fmt.Println("BREAK! phaseStatus=", phaseStatus, " i=", i)
+			break
+		}
+		fmt.Println("phaseStatus=", phaseStatus, " i=", i)
+		time.Sleep(2 * time.Second)
+
+	}
+
+	return result
+
+}
+
+func runPsCommandInWindowsVM(ip, username, password, command string, virtClient kubecli.KubevirtClient, winrmcliPod *k8sv1.Pod) (string, bool) {
+
+	cli := []string{winrmCliCmd, "-hostname", ip, "-username", username, "-password", password}
+	resultCommand := append(cli, command)
+	fmt.Println("command=", resultCommand)
+
+	By(fmt.Sprintf("Running \"%s\" command via winrm-cli", resultCommand))
+
+	output, err := ktests.ExecuteCommandOnPod(virtClient, winrmcliPod, winrmcliPod.Spec.Containers[0].Name, resultCommand)
+	if err != nil {
+		return output, false
+	}
+	return output, true
+
+}
+
+func waitUntilWindowsVMHaveBoot(ip, username, password string, virtClient kubecli.KubevirtClient, winrmcliPod *k8sv1.Pod) bool {
+
+	//ToDo rewrite using API
+	const maxWaitIterations = 60 // two minutes
+	const testCommand = "wmic csproduct get \"UUID\""
+	result := false
+	for i := 0; i < maxWaitIterations; i++ {
+		commandOutput, resultCommand := runPsCommandInWindowsVM(ip, username, password, testCommand, virtClient, winrmcliPod)
+		//Expect(err).ToNot(HaveOccurred())
+		if resultCommand == true {
+			result = true
+			fmt.Println("BREAK! resultCommand=", resultCommand, " i=", i)
+			break
+		}
+		fmt.Println("resultCommand", resultCommand, "commandOutput=", commandOutput, "i=", i)
+		time.Sleep(2 * time.Second)
+
+	}
+
+	return result
+}
+
+func CreatingWinRmiPod(virtClient kubecli.KubevirtClient) *k8sv1.Pod {
+	var winrmcliPod *k8sv1.Pod
+	winrmcliPod = &k8sv1.Pod{
+		ObjectMeta: metav1.ObjectMeta{Name: winrmCli + rand.String(5)},
+		Spec: k8sv1.PodSpec{
+			Containers: []k8sv1.Container{
+				{
+					Name:    winrmCli,
+					Image:   fmt.Sprintf("%s/%s:%s", ktests.KubeVirtRepoPrefix, winrmCli, ktests.KubeVirtVersionTag),
+					Command: []string{"sleep"},
+					Args:    []string{"3600"},
+				},
+			},
+		},
+	}
+	winrmcliPod, err := virtClient.CoreV1().Pods(ktests.NamespaceTestDefault).Create(winrmcliPod)
+	Expect(err).ToNot(HaveOccurred())
+	return winrmcliPod
+
+}
+
+func cleanPV(vmName string) {
```

Comment:
See tests/framework/util.go
Use:
func DeleteResourceWithLabelTestNamespace(resourceType, resourceLabel string)

Reply:
I've deleted it